spec/acceptance/schema_spec.rb

```diff
@@ -60,13 +60,9 @@ class { 'openldap::server': }
 
   context 'remove custom schema' do
     it 'cleans up custom schema' do
-      run_shell('service slapd stop')
-      if os[:family] == 'Debian'
-        run_shell('rm -f /etc/ldap/slapd.d/cn=config/cn=schema/*puppet.ldif')
-      else
-        run_shell('rm -f /etc/openldap/slapd.d/cn=config/cn=schema/*puppet.ldif')
-      end
-      run_shell('service slapd start')
+      shell('service slapd stop')
+      shell('rm -v /etc/ldap/slapd.d/cn=config/cn=schema/*puppet.ldif')
```

Comment:
Why change it from `rm -f` to `rm -v`? Are you expecting it to be present?

Reply:
The file is supposed to be present and `-f` only prevent an error to be thrown if the file is not here.  The `-v` is indeed not needed, it was here to help me try to understand what was going wrong in these tests, while in fact the problem are he result of the tests in the previous file.